src/data/soccer/index.ts

```diff
@@ -34,7 +34,7 @@ export const data: IDataset = {
     .column(buildStringColumn('current_club'))
     .column(buildCategoricalColumn('current_league'))
     .column(buildCategoricalColumn('foot'))
-    .column(buildNumberColumn('height', [0, NaN]))
+    .column(buildNumberColumn('height', [NaN, NaN]))
```

Comment:
why did you change to derived minimum? I mean 0 is the minimal height of a person?